ui/app/components/metric/CurationMetricSelector.tsx

```diff
@@ -76,11 +83,12 @@ export default function CurationMetricSelector<
     return new Set(
       metricsFetcher.data.metrics
         .filter(
-          (m) => !removeDemonstrations || m.metric_name !== "demonstration",
+          (m) =>
+            m.metric_name !== "demonstration" && m.metric_name !== "comment",
```

Comment:
this is wrong. We need to be able to show demontrations in the SFT metric selector. It is causing an e2e test to faill